app/routes/recommendations.tsx

```diff
@@ -0,0 +1,10 @@
+const Recommendations = () => (
+  <div>
+    <p>
+      Recommendation and stuff. On TMDB, Recommendations are get based on one specific movie or tv
+      show, so what will we have here?
```

Comment:
get cả 2 cái rồi gộp lại thành 1 array rồi map render ra :v

Reply:
@Khanhtran47  ý là recommend cho mỗi item ý (tiếng anh của t như shit :v)
[Get a list of recommended movies for a movie.](/movie/{movie_id}/recommendations)
kiểu như vậy chứ nó không có recommend chung chung

Reply:
@ctuanle ừ cái ni thì t nghĩ get đại top trending hay gì đó cũng đc về cho chọn random vài phim rồi render nó lên thôi đc rồi 😂

Reply:
@ctuanle còn recommend movie cho 1 movie thì t nghĩ bỏ vô trang detail của từng movie thôi, kiểu phim tương tự trong mấy trang phim